en/docs/setup/configuring-a-jdbc-user-store.md

```diff
@@ -298,6 +298,14 @@ Eg: Active directory has the MaxPageSize property with the default value of 1000
 Possible values:<br />
 false: Set it to false if the user roles are changed by external means and those changes should be instantly reflected in the Carbon instance.</td>
 </tr>
+<tr class="odd">
+<td>CaseInsensitiveUsername</td>
+<td>properties.CaseInsensitiveUsername</td>
+<td>Case Insensitive Username</td>
+<td>Enables the case insensitivity of the user's username. Default value is ``` true ``` for this configuration. 
+<br />Eg: If a user's username is ``` test ```, that user can also use the username as ``` TEST ```.
```

Comment:
`true`, `test`, and `TEST` should be enclosed in one back tick instead of three otherwise it appears wrong on the site.

Reply:
Fixed in https://github.com/wso2/docs-is/pull/1675 and https://github.com/wso2/docs-is/pull/1679